ocaml/xapi/xapi_cluster_host.ml

```diff
@@ -53,25 +53,58 @@ let sync_required ~__context ~host =
     end
   | _ -> raise Api_errors.(Server_error (internal_error, ["Cannot have more than one Cluster object per pool currently"]))
 
+let call_api_function_with_alert ~__context ~msg ~cls ~obj_uuid ~body ~(api_func : (Rpc.call -> Rpc.response) -> API.ref_session -> 'a) =
+  Helpers.call_api_functions ~__context
+    (fun rpc session_id ->
+       try
+         api_func rpc session_id |> ignore
```

Comment:
I think it is better if `ignore` is not used, and the value is ignored in the caller where we know the actual type.
Otherwise this could mistakenly be a partial application that never runs code at runtime.